index.js

```diff
@@ -85,11 +85,11 @@ function compression (options) {
       }
 
       return stream
-        ? stream.write(toBuffer(chunk, encoding))
+        ? stream.write(toBuffer(chunk, encoding), cb || encoding)
         : _write.call(this, chunk, encoding)
```

Comment:
Shouldn't there be a `cb` here too? So if the web browser does not support compression the `cb` still works instead of it only working if the browser supports compression. A test case for this as well I would guess needs to be added.